interview-questions.md

```diff
@@ -15,54 +15,55 @@ INSTRUCTOR EXAMPLE: What is a conditional statement?
 
 1. What is git? What is the difference between git and Github?
 
-  Your answer:
+  Your answer: Git is a tool/software that allows you to document with your code files with ease on your local machine. Github is a cloud based program that connects git users on the internet.  
 
-  Researched answer:
+  Researched answer: Git is a software that allows users to maintain, document and track their source codes. Github is a git hosting service that connects git users via the internet and allows users to manage their git repositories online.  
 
 
 
 2. Which JavaScript operators will return a Boolean value?
 
-  Your answer:
-
-  Researched answer:
-
+  Your answer: True or false operators will return a Boolean value.
 
+  Researched answer: There are a multitude of Comparison operators that will return of a Boolean value.
```

Comment:
There are three operators that produce that Boolean value of true or false:
Look at Boolean values
https://github.com/learn-academy-2022-bravo/Syllabus/blob/main/javascript/intro.md

Logical (&&, ||, !)
Relational (>, <, <=, >=)
Equality (===, !==)